infra/cardano-sl-infra.cabal

```diff
@@ -234,13 +237,17 @@ test-suite test
                   , cardano-sl-crypto
                   , cardano-sl-crypto-test
                   , cardano-sl-infra
+                  , cardano-sl-networking
                   , cardano-sl-util-test
                   , containers
+                  , dns
                   , generic-arbitrary
                   , hedgehog
                   , hspec
+                  , iproute
                   , kademlia
                   , universum
+                  , unordered-containers
```

Comment:
👍 I was confused where these came from at first, but then realized they are deps from the library which weren't in the test-suite before.